metricbeat/module/apache/status/status.go

```diff
@@ -1,18 +1,35 @@
-// Reads server status from apache host under /server-status?auto mod_status is required.
+// Package status reads Apache HTTPD server status from the mod_status module.
 package status
 
 import (
 	"fmt"
 	"net/http"
 	"net/url"
+	"strings"
 
 	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/logp"
 	"github.com/elastic/beats/metricbeat/helper"
-	_ "github.com/elastic/beats/metricbeat/module/apache"
```

Comment:
I'm happy to remove it here, but then it has to be added to `include/list.go` to make sure the module is still compiled.